ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -670,13 +670,15 @@ if tensorrt_support_is_available
 
   shared_library('nnstreamer_filter_tensorrt',
     filter_sub_tensorrt_sources,
+    #link_args: '-L/usr/local/cuda-12.1/targets/x86_64-linux/lib/stubs',
     dependencies: nnstreamer_filter_tensorrt_deps,
     install: true,
     install_dir: filter_subplugin_install_dir
   )
 
   static_library('nnstreamer_filter_tensorrt',
     filter_sub_tensorrt_sources,
+    #link_args: '-L/usr/local/cuda-12.1/targets/x86_64-linux/lib/stubs',
```

Comment:
Please don't add these commented hardcoded-paths. You can add library-paths when you declare corresponding deps: https://mesonbuild.com/Dependencies.html

Reply:
Done